### DISCUSSION TITLE
2.4ghz or 900mhz for 10km?

### DISCUSSION THREAD

**Discussion by berkeonlu** [ExpressLRS/ExpressLRS#2595] [Q&A]
Hello, I will buy a new expresslrs system.  Remote radiomaster tx16s.  I plan to travel a maximum distance of 15 km.  There are two modules I can purchase right now.  Radiomaster Ranger micro and happmodel es900tx.  Which one do you recommend?

**Comment by 350Chevyman** [comment]
900 mhz requires less mw per unit distance of signal propagation. It also "bends" around objects a little bit better as the wavelength is larger than 2.4 ghz, though the advantage is small. the downsides of 900 mhz are as follows; large rx antennae, low bandwidth, less available channels to fly with others, and less selection of receivers. Most rx and tx are 2.4 ghz. I run 900 mhz as well and accidentally got over a mile out on 10 mw before I realizing I hadn't yet switched my tx module to 2000 mw for distance. (I switch my tx module to 10 mw while testing/configuring my models on the workbench). I'm sure I missed a bunch of stuff as my knowledge of rf is limited but those are some of the basics. Hopefully an rf nerd can come in and explain things more in depth and low-level because I, myself am curious. I should also add that either frequency will get you to 10-15 km without a hitch. Lowering your packet frequency also increases sensitivity to rf. e.g 25 hz will increase rx sensitivy such theoretically you'll still have throughput all the way down to -123 dbm. 50 hz packet rate will decrease sensitivy slightly to -120 dbm for adequate throughput, although these are the theoretical limits and a safety factor of at least +6 dbm should be factored in.

**Comment by aowi7280** [comment]
I have not been able to go more than ~3-3.5km with the Happymodel ES900 (1000mW dynamic, 50Hz packet rate). Mostly due to the video limitation. I just got a new iFlight Chimera and have had failsafes at 2km (200Hz packet rate). I don't know how people are getting 10+km. Would love to hear their secret.

**Reply by pkendall64** [reply]
Failsafes, so you had to walk 2km to get your quad?
Or do you mean you get telemetry lost messages? These are expected and the distance that they happen depends on the RX because of the output power for telemetry on the receiver.

**Reply by aowi7280** [reply]
Not telemetry lost. Rescue/RTH kicked in.

**Reply by pkendall64** [reply]
I would suggest doing a bench test https://www.expresslrs.org/quick-start/pre-1stflight/?h=bench#bench-test
And seeking help on discord in the #help-and-support channel.

**Reply by aowi7280** [reply]
Thanks. I think you nailed it. 2 out of 3 of my antennas are bad. -45db or more at 10mW. The one good one is -20db. It just happened when I took my new quad out for a flight. Coincidence.

**Reply by pkendall64** [reply]
Glad you got it sorted.
